src/http/proxy_wasm/ngx_http_proxy_wasm_dispatch.c

```diff
@@ -150,6 +150,9 @@ ngx_http_proxy_wasm_dispatch(ngx_proxy_wasm_exec_t *pwexec,
     ngx_http_wasm_req_ctx_t         *rctxp = NULL;
     ngx_http_proxy_wasm_dispatch_t  *call = NULL;
     unsigned                         enable_ssl = 0;
+#if (NGX_HAVE_UNIX_DOMAIN)
+    static ngx_str_t                 uds_prefix = ngx_string("unix:");
+#endif
 
```

Comment:
@thibaultcha I'm uncertain where UDS related variables should be declared.

`uds_prefix` is used both by `ngx_http_proxy_wasm_dispatch` and `ngx_http_proxy_wasm_dispatch_request` which makes me want to declare it at the beginning of the file, maybe after `static char  ngx_http_proxy_wasm_host[] = "Host: ";`.

I would like declare it as a `ngx_str_t` to benefit from the `len` attribute later on, but since the other globals that would precede it are actually `char` arrays, it doesn't sound like a good fit either.